.travis.yml

```diff
@@ -2,7 +2,11 @@ language: java
 matrix:
   include:
     - jdk: oraclejdk8
-    
+
+script:
+  - cd test && ./runtests.sh && cd .. # system tests
+  - ./gradlew clean test # JUnit tests
```

Comment:
We can either use `system tests` and `unit/integration tests` OR `I/O tests` and `JUnit tests`?